packages/plugin-core/src/components/views/PreviewPanel.ts

```diff
@@ -346,6 +387,7 @@ export class PreviewPanel implements PreviewProxy, vscode.Disposable {
           data: {
             note,
             syncChangedNote,
+            isLocked: this.isLocked(),
```

Comment:
[`64ae6c4` (#3293)](https://github.com/dendronhq/dendron/pull/3293/commits/64ae6c48bd50fbe461b76a7e42a534f65c93f5cf)

agreed. this is not necessary now since we go for behavoir 1 (reseting lock state after dispose). If we conclude behavior 2 at some point we will need to sync this up with the webview panel but then could just call `sendLockMessage` in `packages/plugin-core/src/components/views/PreviewPanel.ts 149`